docs/releases.rst

```diff
@@ -3,6 +3,100 @@
 Release notes
 =============
 
+Version 1.2
+-----------
+
+This version introduces significant improvements over the handling of synchronous replication, makes the start process and failover more reliable, adds PostgreSQL 9.6 support and fixes plenty of bugs.
+In addition, the documentation, including these release notes, has been moved to https://patroni.readthedocs.org.
+
+**Synchronous replication**
+
+- Add synchronous replication support. (Ants Aasma)
+
+  Adds a new configuration variable ``synchronous_mode``. When enabled Patroni will manage ``synchronous_standby_names`` to enable synchronous replication whenever there are healthy standbys available. When synchronous mode is enabled, Patroni will automatically fail over only to a standby that was synchronously replicating at the time of the master failure. This effectively means zero lost user visible transactions. See the `feature documentation <http://patroni.readthedocs.io/en/latest/replication_modes.html#synchronous-mode>`__ for the detailed description and implementation details.
```

Comment:
This effectively means zero lost user visible transactions. -> This effectively means no user visible transaction gets lost in such a case.